src/common/base/NebulaKeyUtils.cpp

```diff
@@ -11,9 +11,13 @@ namespace nebula {
 // static
 std::string NebulaKeyUtils::vertexKey(PartitionID partId, VertexID vId,
                                       TagID tagId, TagVersion tv) {
+    static int32_t tagMask = 0xBFFFFFFF;
```

Comment:
1. For such constants, it would be better to place them in some well-known place, if they are or will be used in multiple places. 
  2. In my humble opinion, constants used as bit masks should be of type unsigned.
  3. For your information, declaring `tagMask`  like this will occupy memory spaces, in the global data segment. `constexpr` is what should use instead.